### ISSUE TITLE
Add support for proxy

### ISSUE THREAD

**Issue by denics** [macfusion-ng/macfusion2#38]
Dear all, thanks for keeping Macfusion alive (is it?)
I am using it on my High Sierra without any problem (after upgrading manually sshfs), but when I try to check for updates, I am blocked by the proxy.
It is not urgent, but a nice to have I think.
Denis

**Comment by ElDeveloper** [comment]
Unfortunately we have not setup the Sparkle update framework in this fork, hence the updates are likely not working. I've renamed this issue to "Fix sparkle update framework".
